### ISSUE TITLE
Homebrew Formulae/Cask

### ISSUE THREAD

**Issue by kzscisoft** [FAIRDataPipeline/FAIR-CLI#23]
May be worth making a Homebrew formula or even creating a cask for all FAIR projects to allow installs for macOS at some point? @richardreeve

**Comment by richardreeve** [comment]
Yes, absolutely - that would be great if that was possible for those of us on macs once we've got everything working!

**Comment by kzscisoft** [comment]
Also note for Windows:
https://chocolatey.org/#collapsing-right-sidebar

**Comment by richardreeve** [comment]
Cool - I had no idea there was a windows package manager like this. I see there's also https://devblogs.microsoft.com/commandline/windows-package-manager-1-0/ . Presumably you can use a linux package manager if you have WSL installed on Windows... anyway, definitely something for after we submit the paper though!

**Comment by kzscisoft** [comment]
@richardreeve Yep, I think Windows support in general is something which will have to come later too.
